active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+# Background
+> This spec corresponds to [issue 913](https://github.com/microsoft/microsoft-ui-xaml/issues/913) on the WinUI repo.
+
+Users should be informed about essential status changes that occur on an app level.
+These status changes affect the app as a whole and can be either critical or informational.
+Critical status changes like lost internet connectivity are directly impactful to app functionality while informational status changes like an update has completed and been applied are indirectly impactful to app functionality.
+These notifications and corresponding information should be presented in a consistent, predictable, and relevant way to the user depending on the specific scenario.
+
+Currently, Teaching Tip, Content Dialog, and customizations of other flyouts and dialogs exist as options to show these notifications but these controls were not specifically designed to handle app-wide status change notifications. 
+Due to their visual layouts, inherent intrusiveness, or available features they are not sufficient for displaying notifications at an app-wide level.
+
+# Description
+
+An InfoBar is a persistent, actionable, app-wide notification intended for displaying critical and/or actionable information that impact app perception or user experience.
```

Comment:
(Minor comment at this stage) This text is the IntelliSense you'll get from typing <InfoBar>. For Xaml controls it always seems to start with "Represents", and tends to focus more on the 'what'.

```suggestion
Represents an informational control that is visible but does not interfere with the user. 
```